src/main/java/gov/nasa/jpl/mbee/mdk/docgen/docbook/DBTable.java

```diff
@@ -220,17 +220,37 @@ public void transpose() {
                 }
             }
         }
-        this.body = newbody;
         this.headers = null;
+        List<DocumentElement> newColumnHeader = newbody.get(newbody.size() - 1);
```

Comment:
By the looks of it, this is an unrelated fix for a table transposition issue that made its way into here. We typically want to keep our features and fixes independent from one another so we can more easily identify resulting bugs and promptly remedy. We can push forward this time, but noting it here in case it comes up during anyone's debugging and for future reference.